plenum/server/node.py

```diff
@@ -1265,6 +1268,11 @@ def onConnsChanged(self, joined: Set[str], left: Set[str]):
 
         if self.master_primary_name in joined:
             self.lost_primary_at = None
+
+        for inst_id, replica in self.replicas:
```

Comment:
Please, add similar check for case when view_change completed. As of now, during "select_primaries" procedure we can choose already disconnected primary for backup replicas. Therefore, we will have similar problems in this case.
Please, take into account, that initial view_change (by current state) can be done after first "select_primaries" call (catchup by LEDGER_STATUS messages).